rhine-bayes/rhine-bayes.cabal

```diff
@@ -30,10 +30,10 @@ library
     FRP.Rhine.Bayes
   other-modules:
     Data.MonadicStreamFunction.Bayes
-  build-depends:       base         >= 4.11 && < 4.18
```

Comment:
I'd rather keep tight version bounds on `base`, so one should bump to 4.19 here.